package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@api3/data-feed-proxy-combinators",
+  "description": "Contracts to read, combine or adapt data feeds",
+  "keywords": [
+    "api3",
+    "data feed",
+    "oracle",
+    "oev"
+  ],
+  "license": "MIT",
+  "version": "1.0.0",
+  "engines": {
```

Comment:
rm this. It's cumbersome (prevents you to run commands if your node version is has different major version).